src/modules/client/components/clientAlerts/ClientAlert.tsx

```diff
@@ -0,0 +1,52 @@
+import { Alert, AlertTitle, Box, Typography } from '@mui/material';
```

Comment:
(Not related to this line)

I remembered I did add a hack for fixing incorrect ordering for the generated dropdowns.
Try adding `ClientAlertPriorityLevel: ['low'],` to [this object](https://github.com/greenriver/hmis-frontend/blob/e7fbbaae3ecad78b026d93ef61dc3f00844222bc/scripts/generateEnumDescriptions.js#L20-L23) and re-running codegen. That should re-order it correctly!